tests/cli_integration_tests.rs

```diff
@@ -83,7 +83,7 @@ mod no_conversion {
             grex.args(&["-f", file.path().to_str().unwrap()]);
             grex.assert()
                 .success()
-                .stdout(predicate::eq("^(?:b\\\\n|äöü|[ac♥])$\n"));
+                .stdout(predicate::eq("^(?:b\\\\n|äöü|[ac♥])?$\n"));
```

Comment:
Why is the `?` quantifier put here? This would erroneously match the empty string which is not part of the input test cases. Are you able to fix that, too?

Reply:
This one isn't an error. There's two `\n` in a row in the input, so the input includes the empty string. Previously the empty string couldn't be accepted by the dfa.

Reply:
Ah, you are right.